.github/workflows/airbyte-ci-tests.yml

```diff
@@ -88,7 +88,7 @@ jobs:
           gcs_credentials: ${{ secrets.METADATA_SERVICE_PROD_GCS_CREDENTIALS }}
           sentry_dsn: ${{ secrets.SENTRY_AIRBYTE_CI_DSN }}
           github_token: ${{ secrets.GH_PAT_MAINTENANCE_OCTAVIA }}
-          subcommand: "test airbyte-ci/connectors/pipelines --poetry-run-command='pytest tests'"
+          subcommand: "test airbyte-ci/connectors/pipelines --poetry-run-command='pytest tests' --poetry-run-command='mypy pipelines --check-untyped-defs' --poetry-run-command='ruff check pipelines'"
```

Comment:
Just wondering, why `--check-untypted-defs` over `--disallow-untyped-defs`?

Reply:
Nice! Hope that doesn't add too much work to the second pr in the stack 🙈

Reply:
Nope, I fixed everything that failed!